### DISCUSSION TITLE
[REQUEST]Does OpenVDB support ARM structure?

### DISCUSSION THREAD

**Discussion by wangtiehao2010** [AcademySoftwareFoundation/openvdb#1564] [General]
Hello, is it possible to run this project in an Embedded system based on ARM or just support X86?
Looking forward to your help! Thanks a lot!

**Comment by maulanaazhari** [comment]
hi @wangtiehao2010, I am wondering the same thing with you. Have you tried OpenVDB on ARM-based CPU computer? Thanks

**Comment by kmuseth** [comment]
OpenVDB (and certainly NanoVDB) should work just fine on ARM. The only potential challenge is getting all the dependencies (like boost and tbb) to build on ARM, but even that should be fine as long as you have a modern C++ compiler (since all dependencies can be build from source code). For instance, OpenVDB runs on Apples M1/M2 which is ARM :)

Cheers,
Ken

**Comment by malaterre** [comment]
It is a bit challenging to compile openvdb on armel (ARM5) but it seems to work on Debian:

* https://buildd.debian.org/status/logs.php?pkg=openvdb&arch=armel
